### PR TITLE
DOCS-6730 - Add images and configuration text

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by nazarethjim — APPROVED** [review]

File: process-automation/latest/model/connectors/docusign.md [hunk 1]
```diff
@@ -98,6 +98,57 @@ When a Process Automation process is instantiated this way, the following variab
 | envelopeId | string | Envelope ID of the document. |
 | documents | array | Documents related to the envelope and data related to them like uri, and id. |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/docusign.md:100]: The following internal page link is not working: 

`* [DOWNLOAD_DOCUMENT](#download_document)`

File: process-automation/latest/model/connectors/docusign.md [hunk 2]
```diff
@@ -98,6 +98,57 @@ When a Process Automation process is instantiated this way, the following variab
...
+6. Under **Event settings > Trigger events > Recipient Events**, ensure the following are selected:
+
+    * **Recipient Sent**
+    * **Recipient Auto Responded**
+    * **Recipient Delivered**
+    * **Recipient Signed/Completed**
+    * **Recipient Declined**
+    * **Recipient Authentication Failure**
+
+        ![Custom Configuration]({% link process-automation/images/settings.png %})
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/docusign.md:134]: Cannot really see what's in this picture, if it matters

File: process-automation/latest/model/connectors/docusign.md [hunk 3]
```diff
@@ -98,6 +98,57 @@ When a Process Automation process is instantiated this way, the following variab
...
+
+![Logs and failures]({% link process-automation/images/logs-failures.png %})
+
+#### Triggers for DocuSign events
+
+1. From the Modeling Application, create a new trigger with one of the [events](#events).
+
+2. You can filter the trigger using the condition field, which is an expression that must be true in order to trigger an action.
+
+    ![Trigger]({% link process-automation/images/trigger.png %})
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/docusign.md:150]: Not completely clear the picture, if it matters

**Review by gravitonian — CHANGES_REQUESTED** [review]
Just fix one link, I don't know if anything can be done with the large pictures

File: process-automation/latest/model/connectors/docusign.md (hunk 2 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/docusign.md]: Ive removed them all

**Review by gravitonian — APPROVED** [review]
LGTM

**Pull request merged**